front-page.php

```diff
@@ -0,0 +1,348 @@
+<?php
+get_header();
+?>
+
+<!-- main content -->
+
+<main id="main-content" class="container">
+
+
+<?php 
+  // WP_Query arguments
+  $args = array (
+    'category_name'          => 'featured',
+    'posts_per_page'         => '1',
+  );
+
+  // The Query
+  $featured_query = new WP_Query( $args );
+
+
+  // The Loop
+  if ( $featured_query->have_posts() ) { 
+?>
+
+  <section id="featured-post">
+
+<?php
+    while ( $featured_query->have_posts() ) {
+      $featured_query->the_post();
+
+      $featured_id = $post->ID;
+
+      $subtitle = get_post_meta( $post->ID, '_igv_post_subtitle', true );
+?>
+
+    <article <?php post_class(); ?> id="post-<?php the_ID(); ?>">
+
+      <a href="<?php the_permalink() ?>">
+
+        <div class="col s3">
+          <h3 class="featured-title">
+            <span class="rotate-text">
+              <?php the_title(); ?>
+            </span>
+          </h3>
+        </div>
+
+        <?php if ($subtitle) { ?>
+        <div class="featured-subtitle col s3">
+          <span class="rotate-text">
+            <?php echo $subtitle; ?>
+          </span>
+        </div>
+        <?php } ?>
+
+        <div class="col s12">
+          <?php the_post_thumbnail(); ?>
+        </div>
+
+      </a>
+
+    </article>
+
+<?php 
+    }
+?>
+
+  </section>
+
+<?php 
+  } 
+  wp_reset_postdata(); 
+?>
+
+
+  <section class="row"> <!-- Puta portadazza, Noticias, Ads -->
```

Comment:
why? all other loops open section inside the if.
In this case, if the query for puta is empty it will endup with an empty `<section>`